dlc-manager/src/contract/offered_contract.rs

```diff
@@ -44,6 +44,11 @@ pub struct OfferedContract {
     pub cet_locktime: u32,
     /// The time at which the contract becomes refundable.
     pub refund_locktime: u32,
+    /// The denominator `x` for i = `1/x`, where i is the percentage of locked collateral to be paid to the protocol as a fee.
+    /// 0 for no fee
+    pub fee_percentage_denominator: u64,
+    /// The address to which the protocol fee is paid
+    pub fee_address: String,
```

Comment:
Might be better to use `Address` here to fail early if the provided address is not valid

Reply:
Thanks @bonomat, I will try, but I think I needed to use a String at this level because of the way the `impl_dlc_writeable` macro works, and it needs lightning::util::ser::Writeable, which Address does not.